zerocopy-derive/src/lib.rs

```diff
@@ -75,7 +75,8 @@ macro_rules! derive {
     ($trait:ident => $outer:ident => $inner:ident) => {
         #[proc_macro_derive($trait)]
         pub fn $outer(ts: proc_macro::TokenStream) -> proc_macro::TokenStream {
-            let ast = syn::parse_macro_input!(ts as DeriveInput);
+            let mut ast = syn::parse_macro_input!(ts as DeriveInput);
+            ast.generics.make_where_clause();
```

Comment:
Add a comment explaining why we do this?

Reply:
This was vestigial; [removed](https://github.com/google/zerocopy/compare/96b54d1b8190c3b88a8d6d0315e14fd241a5a35a..9844c1f976bafef394033070eaf3cef0cf9e4c58#diff-18618e7fc588bc81a0d47bd0e93c08213abdec19646cff8c8be2b654e3c9872dL533).